doc/03_devops_integration.md

```diff
@@ -0,0 +1,120 @@
+
+    kubectl create secret docker-registry intershop-pull-secret \
+      --context="docker-desktop" \
+      --docker-server=docker.tools.intershop.com \
+      --docker-username='<your Intershop Azure account>' \
+      --docker-password='<CLI secret, see above>'
+
+### Add the pull-secret *intershop-pull-secret* to configuration of devenv-4-iom
+
+Finally, the name of the newly created Kubernetes secret has to be passed to *devenv-4-iom*. To do so, set the key *IMAGE_PULL_SECRET* within the user-specific configuration file of *devenv-4-iom* (devenv.user.properties):
+
+    # change into the root directory of IOM project
+    echo IMAGE_PULL_SECRET=intershop-pull-secret >> devenv.user.properties
+
+## Get Access to *Project Docker Repository*
+
+The *Project Docker Repository* (*Project Docker Repo* in the figure above) contains all project-specific Docker images, which have passed the CI process of the *Azure DevOps Environment*. The Docker images of this repository should be used by *devenv-4-iom* to run local customizations/configurations on top of them.
+
+Since the *Project Docker Repository* is a private Docker registry too, a second *image pull secret* has to be created. For this second *image pull secret* an access token is required. 
+
+### Login into Azure
+
+    # Use the azure-cli Docker image, to avoid local installation of Azure CLI tools.
+    # See https://docs.microsoft.com/en-us/cli/azure/run-azure-cli-docker
+    docker run -it mcr.microsoft.com/azure-cli
+
+    # Login with your Azure account and your Azure password.
+    az login --username <your Intershop Azure account> --password <your Intershop Azure password>
+
+### Get the name of the *Project Docker Repository*
+
+The *Project Docker Repository* is an *Azure Container Registry* (*ACR*). It is named after organization owning the *Azure DevOps Environment*. If you are not sure about the correct name of the ACR, you can get a list of all ACRs within your Azure Subscription. In this case, you need to know your Azure Subscription-ID.
+
+    # Execute the command within running azure-cli Docker container.
+    # Look for a name, that is matching your organization. This name has to be
+    # used in subsequent commands.
+    az acr list --subscription <your Azure Subscription-ID> --out table
+
+### Get the token for accessing the ACR
+
+    # Execute the command within running azure-cli Docker container.
+    # Get the token that's required to access the Docker Registry of your project.
+    # The output provides also the name of the login-server, which is required by the
+    # following command too.
+    az acr login --name <name of the project's ACR, see above> --expose-token
+
+    # end the azure-cli Docker image
+    exit
+
+### Create the Kubernetes secret *project-pull-secret*
+
+The following commands have to be executed outside the azure-cli Docker container.
+
+    kubectl create secret docker-registry project-pull-secret \
```

Comment:
Name should be dynamic, e.g. <registry-name>-pull-secret. Otherwise this leads to problems when working on multiple projects.